specification/operationalinsights/data-plane/Microsoft.OperationalInsights/preview/2021-05-19/OperationalInsights.json

```diff
@@ -0,0 +1,1430 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-05-19_Preview",
+    "title": "Azure Log Analytics",
+    "description": "This API exposes Azure Log Analytics query capabilities",
+    "termsOfService": "https://dev.loganalytics.io/tos",
+    "contact": {
+      "name": "AIAPI Team",
+      "url": "https://dev.loganalytics.io/support",
+      "email": "aiapi@microsoft.com"
+    },
+    "license": {
+      "name": "Microsoft",
+      "url": "https://dev.loganalytics.io/license"
+    }
+  },
+  "host": "api.loganalytics.io",
+  "basePath": "/v1",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
```

Comment:
does this need to specify the target resource as https://api.loganalytics.io?  Similar to this one 

https://github.com/Azure/azure-rest-api-specs/blob/master/specification/applicationinsights/data-plane/Microsoft.Insights/preview/v1/AppInsights.json#L32

That way autorest can generate the correct token?

Reply:
@jamlai - we don't need the exact resource as the resource is provided by the generated client when the auth scope is set. We don't have it set in any other service swaggers.